doc-content/enterprise-only/bpmn/bpmn-support-con.adoc

```diff
@@ -7,117 +7,131 @@ The following table contains a list of icons used to indicate whether a BPMN2 el
+a|Conditional      | image:BPMN2/grn_check.png[]     | image:BPMN2/grn_check.png[]                
+a|Link             |        | image:BPMN2/bk_x.png[]                  
+a|Signal           | image:BPMN2/grn_check.png[]     | image:BPMN2/grn_check.png[]            
+a|Multiple         | image:BPMN2/bk_x.png[]      | image:BPMN2/bk_x.png[]             
+a|Parallel Multiple  | image:BPMN2/bk_x.png[]     | image:BPMN2/bk_x.png[]                
 |===
-h|Element type
-h|Element
-h|Suported
-
-
-|Tasks
-|Business rule 
-|image:BPMN2/grn_check.png[]
-
-
-|
-|Script 
-|image:BPMN2/grn_check.png[]
-
 
+.BPMN2 throwing and non-interrupting events
+[cols="20%,20%,20%,20%,20%" options="header"]
+|===
+|Element Name    
+|Throwing                       
+|                               
+|Non-interrupting             
 |
-|User task
-|image:BPMN2/grn_check.png[]
+|             
+|End                           
+| Intermediate                  
+| Start                       
+| Intermediate
+
+a|None             | image:BPMN2/grn_check.png[]   |                               |                             |
+a|Message          | image:BPMN2/grn_check.png[]   | image:BPMN2/grn_check.png[]   | image:BPMN2/grn_star.png[]  | image:BPMN2/grn_check.png[] 
+a|Timer            |                               |                               | image:BPMN2/bk_x.png[]      | image:BPMN2/grn_check.png[] 
+a|Error            | image:BPMN2/grn_check.png[]   |                               | image:BPMN2/grn_star.png[]  | image:BPMN2/grn_check.png[] 
+a|Escalation       | image:BPMN2/grn_check.png[]   | image:BPMN2/grn_check.png[]   | image:BPMN2/grn_star.png[]  | image:BPMN2/grn_check.png[] 
+a|Cancel           | image:BPMN2/bk_x.png[]                              |                               |                             |   
+a|Compensation     | image:BPMN2/grn_check.png[]   | image:BPMN2/grn_check.png[]   |                             |    
+a|Conditional      |                               |                               | image:BPMN2/grn_star.png[]  | image:BPMN2/grn_check.png[]    
+a|Link             |                               | image:BPMN2/bk_x.png[]        |                             |       
+a|Signal           | image:BPMN2/grn_check.png[]   | image:BPMN2/grn_check.png[]   | image:BPMN2/grn_star.png[]  | image:BPMN2/grn_check.png[] 
+a|Terminate        | image:BPMN2/grn_check.png[]   |                               |                             |       
+a|Multiple         | image:BPMN2/bk_x.png[]        | image:BPMN2/bk_x.png[]        | image:BPMN2/bk_x.png[]      | image:BPMN2/bk_x.png[]     
+a|Parallel Multiple  |                             |                               | image:BPMN2/bk_x.png[]      | image:BPMN2/bk_x.png[]     
+|===
 
-|
-|Custom task
-|image:BPMN2/bk_x.png[]
+.BPMN2 elements
+[cols="33%,33%,34%" options="header"]
+|===
+|Element type
+|Element
+|Supported
 
-|Subprocesses
-|Embedded
-|image:BPMN2/grn_check.png[]
```

Comment:
The shared rows (like the items in first column) would be nice to merge, for cleanliness. Here's an example of how to do that in ascii. More info in the [ascii user guide](https://asciidoctor.org/docs/user-manual/#cell). It's doable in a similar way in the existing format that you have, but don't know off the top of my head. Minor, of course.

```
[cols="33%,33%,34%" options="header"]
|===
|Element type
|Element
|Supported

.4+|Task
|Business rule 
a|
image:BPMN2/grn_check.png[]

|Script
a|image:...

|User task
a|image:...

|Custom task
a|image:...

.4+|Subprocess
|embedded
a|image:...

...

```